packages/a11y/README.md

```diff
@@ -16,20 +16,16 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
-### setup
-
-[src/index.js#L16-L26](src/index.js#L16-L26)
+<a name="setup" href="#setup">#</a> **setup** [\<>](src/index.js#L16-L26)
```

Comment:
I'd be curious if NPM would strip anything of the custom anchor tag. Is there a reason it ought _not_ be a heading, or is that the idea of "toning down"? Structurally / semantically, it seemed to make sense as a heading.

Reply:
Not really, just that this approach made the symbols more prominent. I'm on mobile now but can upload some before/after images.

Reply:
Since it's auto-generated it probably doesn't matter that much whether we use heading. However, I'm curious if we could keep h3 as it helped to give more focus to the method name.